packages/site/src/data/svelte/communities.ts

```diff
@@ -201,14 +201,6 @@ export const communities: Community<(typeof communityTags)[number]>[] = [
 		href: 'https://www.jsday.ie/',
 		tags: ['conferences'],
 	},
-	{
-		name: 'Svelte Summit',
-		description: 'A meetup dedicated to the latest release of Svelte',
-		image: 'https://sveltesociety.dev/images/logo.svg',
-		type: 'Live Events',
-		href: 'https://beta.guild.host/svelte-society-london/events',
-		tags: ['conferences'],
-	},
```

Comment:
Looks outdated as a conference, but appears to be a still-active community. Why remove it? Just needs some updating to make it a bit more evergreen.

Reply:
This was a duplicate and the link is not the correct link for Svelte Summit. 
There is another entry that has the correct info.